notebooks/COS/Extract/Extract.ipynb

```diff
@@ -0,0 +1,1580 @@
+    "\n",
+    "**This tutorial aims to prepare you to work with the COS data of your choice by walking you through altering the extraction box sizes in the XTRACTAB/`_1dx` file to make sure you are extracting the cleanest possible signal from your source and background.** We will demonstrate this in both the NUV and FUV. \n",
+    "\n",
+    "*Note* that some COS modes which use the FUV detector can be better extracted using the [TWOZONE method](https://hst-docs.stsci.edu/cosdhb/chapter-3-cos-calibration/3-2-pipeline-processing-overview), which is not directly discussed in this notebook. All COS/NUV modes use the [BOXCAR method](https://hst-docs.stsci.edu/cosdhb/chapter-3-cos-calibration/3-2-pipeline-processing-overview#id-3.2PipelineProcessingOverview-3.2.1OverviewofTWOZONEextraction) discussed in this notebook.\n",
+    "\n",
+    "- For an in-depth manual to working with COS data and a discussion of caveats and user tips, see the [COS Data Handbook](https://hst-docs.stsci.edu/display/COSDHB/).\n",
+    "- For a detailed overview of the COS instrument, see the [COS Instrument Handbook](https://hst-docs.stsci.edu/display/COSIHB/).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "sunset-oliver",
+   "metadata": {},
+   "source": [
+    "## We will import the following packages:\n",
+    "\n",
+    "- `numpy` to handle arrays and functions\n",
+    "- `astropy.io fits` and `astropy.table Table` for accessing FITS files\n",
+    "- `glob`, `os`, and `shutil` for working with system files\n",
+    "- `astroquery.mast Observations` for finding and downloading data from the [MAST](https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html) archive\n",
+    "- `matplotlib.pyplot` for plotting\n",
+    "- `matplotlib.image` for reading in images\n",
+    "- `calcos` to run the CalCOS pipeline for COS data reduction\n",
+    "- `scipy.interpolate interp1d` for interpolating datasets to the same sampling\n",
+    "\n",
+    "These python packages are installed standard with the the STScI conda distribution. For more information, see our notebook tutorial on [setting up an environment](https://github.com/spacetelescope/COS-Notebooks/blob/master/Setup/Setup.ipynb).\n",
+    "\n",
+    "We'll also filter out two unhelpful warnings about a deprecation and dividing by zero which do not affect our data processing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "municipal-scottish",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import For: array manipulation\n",
+    "import numpy as np\n",
+    "# Import For: reading fits files\n",
+    "from astropy.table import Table\n",
+    "from astropy.io import fits\n",
+    "\n",
+    "# Import For: downloading the data\n",
+    "from astroquery.mast import Observations as Obs\n",
```

Comment:
This is a "soft" guideline (i.e., if you don't like it feel free to ignore it), but that said:

It's mildly discoraged to use `as` in this way on something that you don't expect to use a *lot*, because the feeling is that it makes the code less reusable to "rename" things that already have a name.  So you might drop the `as Obs`, and instead depend on more line-breaking to make the places you're using `Observations` more readable.

Reply:
Interesting! Didn't know that. I can switch that here.